staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/README.md

```diff
@@ -45,4 +46,6 @@ This plugin provides an integration with Azure Active Directory device flow. If
    To sign in, use a web browser to open the page https://aka.ms/devicelogin and enter the code DEC7D48GA to authenticate.
    ```
 
-   * After signing in a web browser, the token is stored in the configuration, and it will be reused when executing next commands.
+   * After signing in a web browser, the token is stored in the configuration, and it will be reused when executing further commands.
+   * The user name in Kubernetes will look like the following: `https://sts.windows.net/TENANT_ID/#username_claim`. If you are using any authorization method you need to give permissions to that user, e.g. by binding the user to a role in the case of RBAC.
```

Comment:
The issuer prefix is actually optional now and can be omitted via the apiserver oidc username claim prefix flag

Reply:
I see. I had not seen issuer prefixing in testing OIDC before, so I thought it was AAD that was including the prefix. Most probably cause I was always using the email claim. Just saw that this is documented at https://kubernetes.io/docs/admin/authentication/#configuring-the-api-server
I'll change this to make it more clear and link out to the source